py4DSTEM/process/phase/utils.py

```diff
@@ -413,7 +413,8 @@ def polar_coordinates(self, x, y):
         """Calculate a polar grid for a given Cartesian grid."""
         xp = self._xp
         alpha = xp.sqrt(x.reshape((-1, 1)) ** 2 + y.reshape((1, -1)) ** 2)
-        phi = xp.arctan2(x.reshape((-1, 1)), y.reshape((1, -1)))
+        # phi = xp.arctan2(x.reshape((-1, 1)), y.reshape((1, -1))) # bug in abtem-legacy and py4DSTEM<=0.14.9
+        phi = xp.arctan2(y.reshape((1, -1)), x.reshape((-1, 1)))
```

Comment:
```suggestion
        phi = xp.arctan2(y.ravel(), x.ravel())
```

Reply:
This needs to be broadcasted, see above.